finagle/h2/src/main/scala/com/twitter/finagle/buoyant/h2/netty4/Netty4DispatcherBase.scala

```diff
@@ -143,7 +143,10 @@ trait Netty4DispatcherBase[SendMsg <: Message, RecvMsg <: Message] {
               case null if id <= closedId.get =>
```

Comment:
I am curious about this case, Does this assume that we can receive frames from streams that are considered closed in Linkerd's "Stream registry" but are still active on the remote peer? Or does this happen immediately after the remote stream is closed and the close frame just reached Linkerd?